app/src/hooks/api/useSamplingSiteApi.ts

```diff
@@ -0,0 +1,31 @@
+import { AxiosInstance } from 'axios';
+import { ICreateSamplingSiteRequest } from 'features/surveys/observations/sampling-sites/SamplingSitePage';
+
+/**
+ * Returns a set of supported api methods for working with search functionality
+ *
+ * @param {AxiosInstance} axios
+ * @return {*} object whose properties are supported api methods.
+ */
+const useSamplingSiteApi = (axios: AxiosInstance) => {
+  /**
+   * Get search results (spatial)
+   *
+   * @return {*}  {Promise<IGetSearchResultsResponse[]>}
+   */
+  const createSamplingSites = async (
+    projectId: number,
+    surveyId: number,
+    samplingSite: ICreateSamplingSiteRequest
+  ): Promise<any> => {
```

Comment:
Nit: avoid `any`

Reply:
create doesn't return anything so updating to void